src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIHandle.cs

```diff
@@ -36,7 +36,7 @@ protected static async Task AuthenticateAsClientAsync(SslStream sslStream, strin
             {
```

Comment:
On line 21, I would also change the comment to:
```C#
/// Exclude TLS 1.3 in TLS-over-TDS modes (TDS 7.4 and below)
```

Reply:
done.